### PR TITLE
Redesign in Kafka tests

### PR DESCRIPTION
**Pull request by jbescos**
I didn't change functionality, but I did some changes in the tests.

There are less connections/disconnections to Kafka. CDI container is running during all the execution of the tests.

### CODE REVIEW

File: tests/integration/kafka/src/test/java/io/helidon/messaging/connectors/kafka/AbstractSampleBean.java [hunk 1]
```diff
@@ -180,7 +180,7 @@ public void onComplete() {
                     .to(new Subscriber<Message<ConsumerRecord<Long, String>>>() {
                         @Override
                         public void onSubscribe(Subscription subscription) {
-                            LOGGER.fine(() -> "onSubscribe()");
+                            LOGGER.fine(() -> "channel5 onSubscribe()");
```
Comment (danielkec) [inline tests/integration/kafka/src/test/java/io/helidon/messaging/connectors/kafka/AbstractSampleBean.java:183]: Channel 5?

Comment (danielkec) [reply tests/integration/kafka/src/test/java/io/helidon/messaging/connectors/kafka/AbstractSampleBean.java:183]: i see sorry

Comment (jbescos) [reply tests/integration/kafka/src/test/java/io/helidon/messaging/connectors/kafka/AbstractSampleBean.java:183]: It is just the name of the class.

**Comment by jbescos** [comment]
I have run this 1944 times and there were no issues.

Previously, I was able to find an issue after running it 370 times.

**Pull request merged**